package.json

```diff
@@ -1,84 +1,92 @@
-    "@wordpress/data": "^6.1.4",
-    "@wordpress/date": "^4.2.3",
-    "@wordpress/dom-ready": "^3.2.3",
-    "@wordpress/edit-post": "^6.17.0",
-    "@wordpress/element": "^4.0.4",
-    "@wordpress/escape-html": "^2.2.3",
-    "@wordpress/eslint-plugin": "^10.0.0",
-    "@wordpress/hooks": "^3.2.2",
-    "@wordpress/html-entities": "^3.2.3",
-    "@wordpress/i18n": "^4.2.4",
-    "@wordpress/icons": "^6.1.1",
-    "@wordpress/keycodes": "^3.2.4",
-    "@wordpress/plugins": "^4.0.6",
-    "@wordpress/url": "^3.3.1",
-    "autoprefixer": "^10.4.0",
-    "babel-jest": "^29.2.2",
-    "commitizen": "^4.2.4",
-    "cross-spawn": "^7.0.3",
-    "cz-conventional-changelog": "^3.3.0",
-    "eslint": "^8.8.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-import": "^2.25.4",
-    "eslint-plugin-jest": "^27.1.3",
-    "eslint-plugin-react": "^7.28.0",
-    "jest": "^27.4.3",
-    "jest-environment-jsdom": "^27.4.3",
-    "postcss": "^8.4.4",
-    "postcss-focus-within": "^5.0.1",
-    "prettier": "npm:wp-prettier@^2.6.2-beta-1",
-    "semantic-release": "^19.0.5",
-    "semantic-release-version-bump": "^1.4.1",
-    "stylelint": "^14.1.0",
-    "stylelint-config-standard": "^30.0.1",
-    "typescript": "^4.5.5",
-    "webpack": "^5.65.0"
-  },
-  "scripts": {
-    "semantic-release": "semantic-release"
-  },
-  "release": {
-    "branches": [
-      "trunk",
-      {
-        "name": "alpha",
-        "prerelease": true
-      }
-    ]
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/Automattic/newspack-scripts.git"
-  },
-  "devDependencies": {
-    "yargs": "^17.3.0"
-  }
+	"name": "newspack-scripts",
+	"version": "1.0.0",
+	"description": "",
+	"bin": {
+		"newspack-scripts": "./bin/newspack-scripts.js"
+	},
+	"author": "",
+	"license": "ISC",
+	"dependencies": {
+		"@commitlint/cli": "^17.8.1",
+		"@commitlint/config-conventional": "^17.8.1",
+		"@rushstack/eslint-patch": "^1.10.3",
+		"@semantic-release/changelog": "^6.0.3",
+		"@semantic-release/git": "^10.0.1",
+		"@testing-library/jest-dom": "^5.17.0",
+		"@testing-library/react": "^13.4.0",
+		"@typescript-eslint/parser": "^7.13.0",
+		"@wordpress/a11y": "^4.0.0",
+		"@wordpress/api-fetch": "^7.0.0",
+		"@wordpress/base-styles": "^5.0.0",
+		"@wordpress/block-editor": "^13.0.0",
+		"@wordpress/blocks": "^13.0.0",
+		"@wordpress/components": "^28.0.0",
+		"@wordpress/compose": "^7.0.0",
+		"@wordpress/data": "^10.0.0",
+		"@wordpress/date": "^5.0.0",
+		"@wordpress/dom-ready": "^4.0.0",
+		"@wordpress/edit-post": "^8.0.0",
+		"@wordpress/element": "^6.0.0",
+		"@wordpress/escape-html": "^3.0.0",
+		"@wordpress/eslint-plugin": "^19.0.0",
+		"@wordpress/hooks": "^4.0.0",
+		"@wordpress/html-entities": "^4.0.0",
+		"@wordpress/i18n": "^5.0.0",
+		"@wordpress/icons": "^10.0.0",
+		"@wordpress/keycodes": "^4.0.0",
+		"@wordpress/plugins": "^7.0.0",
+		"@wordpress/scripts": "^28.0.0",
+		"@wordpress/url": "^4.0.0",
+		"ajv": "^8.16.0",
```

Comment:
I've had success testing without this dependency (and its `overrides` below) explicitly listed (see https://github.com/Automattic/newspack-scripts/commit/7c386fad81341caccc10c2f54fabb9e799c3713b). Can be removed if your results match.

Reply:
This seems resolved for now, but every now and then if I rerun `npm i` it brings back the "wrong" ajv version. So it's something to look out for in the future.